src/ducks/apps/components/ConfigureModal.jsx

```diff
@@ -33,7 +35,7 @@ export class ConfigureModal extends Component {
         mobileFullscreen
         overflowHidden
         size="small"
-        height="35rem"
+        height={!isMobile && '35rem'}
```

Comment:
Is there absolutely no way to handle this with CSS ?

Reply:
We will have to use a static media query matching the mobile breaking point and target a new class on the modal wrapper here.

Reply:
It does not scare me actually and I am pretty much convinced that it is what we should do.

Any opinion about this @GoOz, @ptbrowne ?

Reply:
No I think this is OK. It is still a patch.

For me the best way for these problems to go away is for the intent to tell the app what its height ? This way the modal would automatically be the right height.

See my comment : https://github.com/cozy/cozy-ui/issues/668#issuecomment-434641325

Reply:
Well it was discussed in the referenced issue.

Reply:
Ok then, so it’s an interapp issue and we agree on the fact that this magical 35rem value should not be here.